src/components/molecules/profile-panel.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+import { Avatar, Button, Panel } from "..";
+
+export const ProfilePanel = () => {
+  const userProfile = {
+    Status: "Active",
+    Address: "1234 Main St, New York, NY 10001",
+    Phone: "012-345-6789",
+    Email: "jane@codehesion.co.za",
+  };
+  const getValues = (obj: Object) => {
```

Comment:
We don't use these types.

'String', 'Object', 'Array'. They dont provide use with any information

Rather type out the entire object.

{
  Status: string;
  Address: string;
  ...
}